tests/test_repro.py

```diff
@@ -1404,3 +1393,58 @@ def test(self):
 
         expected_metrics_display = "{}: {}".format(metrics_file, metrics_value)
         self.assertIn(expected_metrics_display, self._caplog.text)
+
+
+# Pytest style tests
+
+
+@pytest.fixture
+def foo_copy(repo_dir, dvc):
+    stages = dvc.add(repo_dir.FOO)
+    assert len(stages) == 1
+    foo_stage = stages[0]
+    assert foo_stage is not None
+
+    fname = "foo_copy"
+    stage_fname = fname + ".dvc"
+    dvc.run(
+        fname=stage_fname,
+        outs=[fname],
+        deps=[repo_dir.FOO, repo_dir.CODE],
+        cmd="python {} {} {}".format(repo_dir.CODE, repo_dir.FOO, fname),
+    )
+    return {"fname": fname, "stage_fname": stage_fname}
+
+
+def test_dvc_formatting_retained(dvc, foo_copy):
+    root = Path(dvc.root_dir)
```

Comment:
Love this class! Has not been aware about its existence. I assume it is cross platform, right? I wonder if it would slow down the tool if migrate some stuff internally.

Reply:
This is stdlib in Python 3 and a backport in Python 2, so it should work everywhere. Not sure about performance, should be negligible comparing to file access though.